src/Assessment/Processor/ProcessDatasets.cs

```diff
@@ -2246,6 +2251,152 @@ private bool Create_VM_Opportunity_AsOnPrem_Model(List<VM_Opportunity_AsOnPrem>
             UserInputObj.LoggerObj.LogInformation("Updated Business_Case excel model");
         }
 
+        private void Process_Financial_Summary_Model(List<Financial_Summary> Financial_Summary_List, List<SQL_MI_PaaS> SQL_MI_PaaS_List, List<SQL_IaaS_Instance_Rehost_Perf> SQL_IaaS_Instance_Rehost_Perf_List, List<SQL_IaaS_Server_Rehost_Perf> SQL_IaaS_Server_Rehost_Perf_List,
+                                          List<WebApp_PaaS> WebApp_PaaS_List, List<WebApp_IaaS_Server_Rehost_Perf> WebApp_IaaS_Server_Rehost_Perf_List,
+                                          List<VM_IaaS_Server_Rehost_Perf> VM_IaaS_Server_Rehost_Perf_List)
+        {
+            UserInputObj.LoggerObj.LogInformation("Creating excel model for Financial_Summary");
+            if (!AzurePaaSCalculator.IsCalculationComplete())
+            {
+                AzurePaaSCalculator.SetParameters(SQL_MI_PaaS_List, WebApp_PaaS_List);
+                AzurePaaSCalculator.Calculate();
+            }
+
+            if (!AzureIaaSCalculator.IsCalculationComplete())
+            {
+                AzureIaaSCalculator.SetParameters(SQL_IaaS_Instance_Rehost_Perf_List, SQL_IaaS_Server_Rehost_Perf_List, WebApp_IaaS_Server_Rehost_Perf_List, VM_IaaS_Server_Rehost_Perf_List);
+                AzureIaaSCalculator.Calculate();
+            }
+
+            Financial_Summary PaaSWebAppDev = new Financial_Summary();
+            PaaSWebAppDev.MigrationStrategy = "Modernize/Re-Platform(PaaS)";
+            PaaSWebAppDev.Workload = "ASP.NET WebApps on IIS - Dev/Test";
+            PaaSWebAppDev.SourceCount = AzurePaaSCalculator.GetWebAppPaaSDevMachineIdCount();
+            if (GetEnvironmentToPlanCountMap(WebApp_PaaS_List).ContainsKey("Dev")) 
+                PaaSWebAppDev.TargetCount = GetEnvironmentToPlanCountMap(WebApp_PaaS_List)["Dev"];
+            else 
+                PaaSWebAppDev.TargetCount = 0;
+            PaaSWebAppDev.StorageCost = AzurePaaSCalculator.GetWebAppPaaSStorageCost() * 12.0;

```

Comment:
nit: I know, webapp paas storage costs are 0, but if possible, create a function to return those 0 values and variables, separately for dev and prod, instead of using the Same GetWebAppPaaSStorageCost() for both dev and prod.